src/cljs/main/broadfcui/page/workspace/method_configs/method_config_editor.cljs

```diff
@@ -382,7 +420,7 @@
                                                          [:methodConfiguration :methodRepoMethod key]))
                                                [:methodNamespace :methodName :methodPath :sourceRepo])
            config-namespace+name (select-keys (get-in @state [:loaded-config :methodConfiguration])
-                                              [:namespace :name])
+                                              [:namespace :name :rootEntityType])
```

Comment:
Preserve the existing `rootEntityType`. Otherwise we end up with the static and meaningless `"rootEntityType"` (returned from [here](https://github.com/broadinstitute/rawls/blob/develop/core/src/main/scala/org/broadinstitute/dsde/rawls/jobexec/MethodConfigResolver.scala#L143)).